fiftyone.devicedetection.cloud/examples/cloud/configurator-console/configurator.js

```diff
@@ -34,12 +34,18 @@ Required npm Dependencies:
 - fiftyone.devicedetection
 */
 const path = require('path');
+const require51 = (requestedPackage) => {
+  try {
+    return require(path.join(__dirname, '/../../../node_modules/', requestedPackage));
+  } catch (e) {
+    return require(path.join(__dirname, '/../../../../', requestedPackage));
+  }
+};
 
-const DeviceDetectionCloudPipelineBuilder =
-  require(path.join(__dirname, '/../../../deviceDetectionCloudPipelineBuilder'));
+const DeviceDetectionCloudPipelineBuilder = require51('fiftyone.devicedetection.cloud').DeviceDetectionCloudPipelineBuilder;
 const ExampleUtils = require(path.join(__dirname, '/../exampleUtils'));
 
-const run = async function (resourceKey, output) {
+const run = async function (resourceKey = "AQRb8GIR-KZ2oBJg20g", output) {
```

Comment:
resource key hardcoded - it shouldn't be here, it should be taken from the environment variable